Dynatrace/dynatrace-operator
# Update `controller-gen` to `v0.9.0`

# Description
CRD was generated using an outdated version of `controller-gen`.

Removed `+kubebuilder:validation:MaxProperties=1` annotation for `OneAgentSpec` which as incompatible.

## How can this be tested?
Upgrade `controller-gen` and check if the generated CRD is the same:
1. `go install sigs.k8s.io/controller-tools/cmd/controller-gen@v0.9.0`
2. `make manifests`


## Checklist
